test/parallel/test-util-inspect.js

```diff
@@ -1516,3 +1516,31 @@ assert.strictEqual(util.inspect('"\'${a}'), "'\"\\'${a}'");
   value.foo = 'bar';
   assert.notStrictEqual(util.inspect(value), expected);
 });
+
+// Check for special colors.
+{
+  const { inspect } = util;
+  const special = inspect.colors[inspect.styles.special];
+  const string = inspect.colors[inspect.styles.string];
+
+  assert.strictEqual(
+    inspect(new WeakSet(), { colors: true }),
+    `WeakSet { \u001b[${special[0]}m<items unknown>\u001b[${special[1]}m }`
+  );
+  assert.strictEqual(
+    inspect(new WeakSet(), { colors: true }),
```

Comment:
Is this assertion repeated on purpose?

Reply:
removing `author ready` label because of this open question

Reply:
Thanks! Copy paste error :) It's now resolved.